ironfish-rust/Cargo.toml

```diff
@@ -42,6 +42,7 @@ chacha20poly1305 = "0.9.0"
 crypto_box = { version = "0.8", features = ["std"] }
 ff = "0.12.0"
 group = "0.12.0"
+hex-literal = "0.4"
```

Comment:
Should this be a dev-dependency?

Reply:
Yes, oops, not sure how it ended up there